run_ci.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+# Required for libsass to not shit itself in Nix
+bundle config build.sassc --disable-lto
+bundle install
+bundle exec rake db:create db:schema:load
+USE_SELENIUM=1 bundle exec rspec spec
```

Comment:
We could flip the conditional in `feature_spec_dependencies.rb`, drop the env var, and require `USE_POLTERGEIST=true` for those that still want to use PhantomJS. I use selenium exclusively myself.

Reply:
Works for me, but not desperately needed.

Reply:
The third way is to have `SOME_ENV_VAR=selenium` or `SOME_ENV_VAR=poltergeist` and let people commit to their own choice with an envrc or similar, but I'll get back in my bikeshed now :-)